nncf/quantization/advanced_parameters.py

```diff
@@ -130,6 +130,20 @@ class AdvancedSmoothQuantParameters:
     matmul: float = 0.95
 
 
+class RestoreMode(Enum):
+    """
+    Specifies how to revert operations to their original precision.
+
+    :param ACTIVATIONS_AND_WEIGHTS: Operations will be reverted to floating-point precision.
+    :param ONLY_ACTIVATIONS: Certain operations (such as MatMul and Embedding) will
```

Comment:
It looks like you need update the docstring taking into account https://github.com/openvinotoolkit/nncf/pull/2299#discussion_r1432667080

Reply:
Thanks! Fixed.